docs/cloud/grafana.md

```diff
@@ -0,0 +1,29 @@
+# Grafana
+
+GreptimeCloud service can be configured as a [Grafana data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+## Prometheus
+
+Click the `Add data source` button and select `Prometheus` as the type.
+
+Fill in the following information in `HTTP`:
+
+- Prometheus server URL: `https://<host>/v1/prometheus/read?db=<dbname>`
+
+Click `basic auth` in the `Auth` section and fill in the following information in `Basic Auth Details`:
+
+- User: `<username>`
```

Comment:
Is *Your GreptimeCloud service username* better? May this mislead user to fill in the Greptime Cloud username?

Reply:
`<username>` will be replaced automatically at GreptimeCloud console. Maybe I can add some explanation bellow.